trace-statistics-info.md

```diff
@@ -0,0 +1,7 @@
+## Show Statistics Infomation on Trace Query
+
+### Target
+
+​    Show statistics data of in during trace query
+
+![](./new issue/skywalking-trace-statc.png)
```

Comment:
You don't have to add this, and also png file. We don't need to increase the package size.

Reply:
Ok，should commit a delete changeset?

Reply:
Yes, it is fine.

Reply:
This is still not removed.